README.md

```diff
@@ -57,9 +57,10 @@ ETHEREUM_ADDRESS=<your ethereum address>
 READ_ONLY_PROVIDER=http://localhost:8545
 LOCKSMITH_URI=http://localhost:8080
 WEDLOCKS_URI=http://localhost:1337
-DASHBOARD_URL=http://localhost:3000
+DASHBOARD_URL=http://localhost:3000/dashboard
```

Comment:
Again, I didn't write the code. This is just updating the docs to be consistent. Currently, the suggested values simply don't work.

This is a perfect candidate for opening up a follow-up issue. I won't be doing this fix in this PR, as each PR should do only "one thing."

Reply:
I think `DASHBOARD_URL` is actually right and we should not change it. 
I like your idea of using better names such as with `_APP` but for now let;s not change this and just re-use `DASHBOARD_URL` with the right path appended.

Reply:
again, `DASHBOARD_URL` is incorrect. Here is the proof:

The CircleCI env variables are:

<img width="1260" alt="Screen Shot 2019-07-08 at 3 07 52 PM" src="https://user-images.githubusercontent.com/98250/60835871-36e84680-a192-11e9-92dd-0bc1b5cb6f48.png">
<img width="1260" alt="Screen Shot 2019-07-08 at 3 07 31 PM" src="https://user-images.githubusercontent.com/98250/60835872-36e84680-a192-11e9-8c1c-0b55fd9d94ff.png">

Notice they end with `ard/` and not `.com`

and when `DASHBOARD_URL` is set to `http://localhost:3000` here is what happens when running `unlock-protocol.com/`. We start in `localhost:3002` and after accepting terms, we get taken back to the homepage at `http://localhost:3000` instead of to the dashboard at `http://localhost:3000/dashboard`:

![out](https://user-images.githubusercontent.com/98250/60835929-5bdcb980-a192-11e9-9818-be700237ffba.gif)

In order to re-use `DASHBOARD_URL`, the paywall would have to strip `dashboard/` from the path.

Reply:
Please use http://localhost:3000/